Backend/WebAPI/Controllers/UsersController.cs

```diff
@@ -65,6 +71,45 @@ public string Register([FromBody] RegisterRequest request)
             return "got to end without stuff";
         }
 
+        /// <summary>
+        /// NOT DONE. STILL IN PROCESS
+        /// </summary>
+        /// <param name="request"></param>
+        /// <returns></returns>
+        [HttpPost]
+        [Route("api/users/login")]
+        public IHttpActionResult Login([FromBody] LoginRequest request)
+        {
+            IUserService _userService = new UserService();
+            IPasswordService _passwordService = new PasswordService();
+
+            using (var _db = new DatabaseContext())
```

Comment:
This logic can be moved to the manager layer. If we feel like you can do it by the end of the milestone, go for it. If not, do it after so we can get functionality out of the way.

Reply:
It's done. Can you check it? @alfredovargasjr